evap/staff/tests/test_views.py

```diff
@@ -156,7 +156,7 @@ def test_reward_points_granting_message(self):
         student.refresh_from_db()
 
         self.assertIn("Successfully updated user.", page)
-        self.assertIn("The removal of courses has granted the user &quot;{}&quot; reward points for the active semester.".format(student.username), page)
+        self.assertIn("The removal of courses has granted the user &quot;{}&quot; 3 reward points for the active semester.".format(student.username), page)
```

Comment:
please either take the number from the settings file or overwrite settings for this test if you test against a specific value.